### DISCUSSION TITLE
How to access cider 2.x

### DISCUSSION THREAD

**Discussion by ShivamRawat0l** [ciderapp/Cider#1606] [Q&A] [answered]
@cryptofyre As mentioned in the last discussion I am unsure how to access cider 2.x. I checked other branchs and it has 1.6.0. 

Kindly clearify.

**Comment by cryptofyre** [comment, marked as answer]
Currently closed source as we get closer to release we'll begin open sourcing and allowing more people to try it out.

**Reply by lunaisnotaboy** [reply]
Any updates on this?
